src/www/vpn_openvpn_client.php

```diff
@@ -882,7 +887,7 @@ function addRow() {
 <?php
               if (!isset($config['cert']) || count($config['cert']) == 0) :?>
                 <b><?=gettext("No Certificates defined.");?></b> <br /><?=gettext("Create one under");?>
-                <a href="system_certmanager.php"><?=gettext("System: Certificates");?></a> <?=gettext("if one is required for this connection.");?>
+                <a href="/ui/trust/certificates/"><?=gettext("System: Certificates");?></a> <?=gettext("if one is required for this connection.");?>
```

Comment:
Isn't that "System:Trust"? Needs probably more work to do because the string needs to be a template.